README.md

```diff
@@ -157,6 +157,7 @@ H3 has a concept of composable utilities that accept `event` (from `eventHandler
 - `getSession(event, { password, name?, cookie?, seal?, crypto? })`
 - `updateSession(event, { password, name?, cookie?, seal?, crypto? }), update)`
 - `clearSession(event, { password, name?, cookie?, seal?, crypto? }))`
+- `handleCors(options)` (see [h3-cors](https://github.com/NozomuIkuta/h3-cors) for more detail)
```

Comment:
Let me consider it in another time how we can organize README, because it need lots of words to explain CORS features. Or, we might want to create a Docus website for h3.

Reply:
Indeed. If possible adding help in JSDocs would be the best. In the future we should generate README baased on JSDocs and add more. Link is good for me until then 👍🏼